packages/devtools_app/lib/src/code_size/stub_data/app_size.dart

```diff
@@ -0,0 +1,3 @@
t":"T","value":52},{"n":"Type Test Type: Frame*","k":"s","t":"T","value":52},{"n":"Type Test Type: Trace*","k":"s","t":"T","value":48},{"n":"_iso_stub_SlowTypeTestStub","k":"s","t":"T","value":240},{"n":"_iso_stub_UnreachableTypeTestStub","k":"s","t":"T","value":4},{"n":"_iso_stub_TopTypeTypeTestStub","k":"s","t":"T","value":4},{"n":"_iso_stub_DefaultNullableTypeTestStub","k":"s","t":"T","value":24},{"n":"_iso_stub_DefaultTypeTestStub","k":"s","t":"T","value":8},{"n":"_iso_stub_CallClosureNoSuchMethodStub","k":"s","t":"T","value":176},{"n":"_iso_stub_InitLateFinalInstanceFieldStub","k":"s","t":"T","value":156},{"n":"_iso_stub_InitLateInstanceFieldStub","k":"s","t":"T","value":108},{"n":"_iso_stub_InitInstanceFieldStub","k":"s","t":"T","value":60},{"n":"_iso_stub_InitStaticFieldStub","k":"s","t":"T","value":56},{"n":"_iso_stub_InstanceOfStub","k":"s","t":"T","value":72},{"n":"_iso_stub_AssertBooleanStub","k":"s","t":"T","value":40},{"n":"_iso_stub_ReThrowStub","k":"s","t":"T","value":44},{"n":"_iso_stub_ThrowStub","k":"s","t":"T","value":40},{"n":"_iso_stub_ArrayWriteBarrierStub","k":"s","t":"T","value":464},{"n":"_iso_stub_WriteBarrierWrappersStub","k":"s","t":"T","value":192},{"n":"_iso_stub_CloneContextStub","k":"s","t":"T","value":212},{"n":"_iso_stub_AllocateObjectParameterizedStub","k":"s","t":"T","value":100},{"n":"_iso_stub_AllocateObjectStub","k":"s","t":"T","value":80},{"n":"_iso_stub_AllocateContextStub","k":"s","t":"T","value":208},{"n":"_iso_stub_AllocateArrayStub","k":"s","t":"T","value":272},{"n":"_iso_stub_StackOverflowSharedWithoutFPURegsStub","k":"s","t":"T","value":56},{"n":"_iso_stub_StackOverflowSharedWithFPURegsStub","k":"s","t":"T","value":192},{"n":"_iso_stub_AllocateMintSharedWithoutFPURegsStub","k":"s","t":"T","value":112},{"n":"_iso_stub_AllocateMintSharedWithFPURegsStub","k":"s","t":"T","value":248},{"n":"_iso_stub_RangeErrorSharedWithoutFPURegsStub","k":"s","t":"T","value":48},{"n":"_iso_stub_RangeErrorSharedWithFPURegsStub","k":"s","t":"T","value":120},{"n":"_iso_stub_NullArgErrorSharedWithoutFPURegsStub","k":"s","t":"T","value":40},{"n":"_iso_stub_NullArgErrorSharedWithFPURegsStub","k":"s","t":"T","value":112},{"n":"_iso_stub_NullErrorSharedWithoutFPURegsStub","k":"s","t":"T","value":40},{"n":"_iso_stub_NullErrorSharedWithFPURegsStub","k":"s","t":"T","value":112},{"n":"_iso_stub_DispatchTableNullErrorStub","k":"s","t":"T","value":28},{"n":"_iso_stub_BuildMethodExtractorStub","k":"s","t":"T","value":300},{"n":"Type Test Type: _ThumbPressGestureRecognizer*","k":"s","t":"T","value":52},{"n":"Type Test Type: _CupertinoTextSelectionToolbarItemsSlot*","k":"s","t":"T","value":52},{"n":"Type Test Type: _DecorationSlot*","k":"s","t":"T","value":52},{"n":"Type Test Type: _SyncIterator<RenderBox*>*","k":"s","t":"T","value":112},{"n":"Type Test Type: _DropdownMenuRouteLayout<_DropdownMenuRouteLayout::T*>*","k":"s","t":"T","value":88},{"n":"Type Test Type: _HashSetEntry<Layer*>?","k":"s","t":"T","value":96},{"n":"Type Test Type: 
```

Comment:
nit: pretty print this multi-line json so it is more readable. there are a bunch of websites that will do json pretty printing or you can use the dart library method for it.